internal/http/interceptors/log/log.go

```diff
@@ -55,7 +55,7 @@ func (h loggingHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	logger := makeLogger(w)
 	url := *req.URL
 	h.handler.ServeHTTP(logger, req)
-	writeLog(log, req, url, t, logger.Status(), logger.Size())
+	writeLog(log, req, url, t, logger.Status(), logger.Size(), w.Header())
```

Comment:
it should be `logger.Header()` and not `w.Header()`.

The logger records the response before is flushed out, so we need to take the response headers from there.